src/scm/plams/interfaces/adfsuite/ams.py

```diff
@@ -1222,11 +1222,7 @@ def get_ir_intensities(self, engine: Optional[str] = None) -> np.ndarray:
 
         The *engine* argument should be the identifier of the file you wish to read. To access a file called ``something.rkf`` you need to call this function with ``engine='something'``. The *engine* argument can be omitted if there's only one engine results file in the job folder.
         """
-        return np.asarray(
-            self._process_engine_results(lambda x: x.read_reals("Vibrations", "Intensities[km/mol]"), engine)
-        ).reshape(
-            -1,
-        )
+        return np.asarray(self.readrkf("Vibrations", "Intensities[km/mol]", file=engine or "engine")).reshape((-1,))
```

Comment:
To stop mypy errors, you will need a `cast` here as per: https://github.com/SCM-NV/PLAMS/pull/322/files#diff-81211846ed4f0a3485470ee86c04d7c73fcb4466aac2829f9f86eefd4b36e5c9R429